package.json

```
@@ -7,6 +7,9 @@
   "license": "MIT",
   "main": "dist/server.js",
   "type": "module",
+  "engines": {
```

Biome (2.5.7)

[error] The key engines was already declared.

(lint/suspicious/noDuplicateObjectKeys)